snapshots/snapshots-annotations/build.gradle.kts

```diff
@@ -23,31 +21,19 @@ java {
 }
 
 tasks.withType<KotlinCompile> {
-  kotlinOptions {
-    jvmTarget = JavaVersion.VERSION_11.toString()
-    languageVersion = "1.7"
+  compilerOptions {
+    jvmTarget = JvmTarget.JVM_11
+    languageVersion.set(KotlinVersion.KOTLIN_1_8)
   }
 }
 
 dependencies {
   implementation(libs.kotlinx.serialization)
 }
 
-tasks.register("generateMetaInfVersion") {
-  doLast {
-    metaInfDestDir.mkdirs()
-    File(metaInfDestDir, "version.txt").writeText(
-      "version: $version" +
-        "\nrevision: ${grgit.head().id}"
-    )
-  }
-}
-
-afterEvaluate {
-  tasks.filter { task ->
-    task.name.startsWith("generate") && task.name.endsWith("Resources")
-  }.forEach { task ->
-    task.dependsOn(tasks.findByName("generateMetaInfVersion"))
+tasks.withType<Jar> {
+  manifest {
+    attributes["Implementation-Version"] = version
```

Comment:
The version is now available a meta-inf attribute called “Implementation-Version”. 
Does this break anything?

Reply:
Yeah this will break quite a few things as we rely on this version field when checking on the backend. Can you leave `version:` as-is and you can add the `Implementation-Version` which we can migrate to?

Reply:
Chatted offline, misunderstood the change so this should be all good since this is not affecting the library module!